components/chart/types/waterfall.md

```diff
@@ -0,0 +1,179 @@
+            },
+            new CashFlowData
+            {
+                Period = "Jan",
+                Amount = 17000,
+                Color = "green"
+            },
+            new CashFlowData
+            {
+                Period = "Feb",
+                Amount = 14000,
+                Color = "green"
+            },
+            new CashFlowData
+            {
+                Period = "Mar",
+                Amount = -12000,
+                Color = "red"
+            },
+            new CashFlowData
+            {
+                Period = "Q1",
+                Summary = "runningTotal",
+                Color = "gray"
+            },
+            new CashFlowData
+            {
+                Period = "Apr",
+                Amount = -22000,
+                Color = "red"
+            },
+            new CashFlowData
+            {
+                Period = "May",
+                Amount = -18000,
+                Color = "red"
+            },
+            new CashFlowData
+            {
+                Period = "Jun",
+                Amount = 10000,
+                Color = "green"
+            },
+            new CashFlowData
+            {
+                Period = "Q2",
+                Summary = "runningTotal",
+                Color = "gray"
+            },
+            new CashFlowData
+            {
+                Period = "Ending\nBalance",
+                Summary = "total",
+                Color = "#555"
+            }
+        };
+    }
+
+    private class CashFlowData
+    {
+        public string Period { get; set; }
+        public decimal? Amount { get; set; }
+        public string Summary { get; set; }
+        public string Color { get; set; }
+    }
+}
+````
+
+## Waterfall Chart Specific Appearance Settings
+
+### Orientation
+
+There are two types of Waterfall charts: 
+ * Traditional Waterfall Chart - displays changes vertically. To use this Waterfall Chart, you need to set the series type to `ChartSeriesType.Waterfall`.
+![vertical waterfall chart](images/vertical-waterfall-chart.png)
+ * Horizontal waterfall chart - presents changes horizontally. To use this type of Waterfall Chart, you need to configure the series type as `ChartSeriesType.HorizontalWaterfall`.
+![horizontal waterfall chart](images/horizontal-waterfall-chart.png)
+
+### Labels
+
+Each data item is decorated with a text label. You can control and customize them through the `<ChartCategoryAxisLabels />` and its children tags.
+
+* `Visible` - hide all labels by setting this parameter to `false`.
+* `Step` - renders every n-th labels, where n is the value(double number) passed to the parameter.
+* `Skip` - skips the first n labels, where n is the value (double number) passed to the parameter.
+* `Angle` - rotates the labels with the desired angle by n degrees, where n is the value passed to the parameter. It can take positive and negative numbers. To set this parameter use the `< ChartCategoryAxisLabelsRotation />` child tag.
```

Comment:
```suggestion
* `Angle`—Rotates the labels with the desired angle by n degrees, where n is the value passed to the parameter. It can take positive and negative numbers. To set this parameter, use the `< ChartCategoryAxisLabelsRotation />` child tag.
```